pygfx/utils/transform.py

```diff
@@ -573,9 +568,9 @@ def position(self) -> np.ndarray:
     def position(self, value):
         if self.state_basis == "components":
             self._position[:] = value
-            self.flag_update()
-            return
-        AffineBase.position.fset(self, value)
+        else:
+            la.mat_compose(value, self.rotation, self.scale, out=self._matrix)
+        self.flag_update()
```

Comment:
Does it make sense to also update the `AffineBase.position` setter to use `la.mat_compose(... out=self._matrix)`?

Reply:
No because self_matrix doesn't exist there